bundles/binding/org.openhab.binding.fritzboxtr064/.classpath

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" output="target/classes" path="src/main/java"/>
+	<classpathentry excluding="**" kind="src" output="target/classes" path="src/main/resources"/>
+	<classpathentry exported="true" kind="lib" path="lib/commons-codec-1.9.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/commons-logging-1.2.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/fluent-hc-4.4.1.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/httpclient-cache-4.4.1.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/httpclient-win-4.4.1.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/httpmime-4.4.1.jar"/>
```

Comment:
is httpclient 4.4.1 really necessary? The current target platform comes with the 4.1.3 which might be sufficient as well?